src/Entity/ItemizedPaymentInformation.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Ups\Entity;
+
+use LogicException;
+
+class ItemizedPaymentInformation
+{
+
+    /**
+     * @var splitDutyVATIndicator
+     */
+    private $splitDutyVATIndicator;
+    private $transportationShipmentCharge = null;
```

Comment:
`null` is not necesarry here (it's the default). Could you also add a docblock to this property + the one below?